app/src/test/java/org/simple/clinic/registration/facility/RegistrationFacilitySelectionScreenControllerTest.kt

```diff
@@ -46,14 +47,14 @@ class RegistrationFacilitySelectionScreenControllerTest {
   @get:Rule
   val rxErrorsRule = RxErrorsRule()
 
-  private val uiEvents = PublishSubject.create<UiEvent>()!!
+  private val uiEvents = PublishSubject.create<UiEvent>()
   private val screen = mock<RegistrationFacilitySelectionScreen>()
   private val facilitySync = mock<FacilitySync>()
   private val facilityRepository = mock<FacilityRepository>()
   private val userSession = mock<UserSession>()
   private val currentTime = Instant.parse("2018-01-01T00:00:00Z")
   private val utcClock = TestUtcClock(currentTime)
-  private val listItemBuilder = mock<FacilityListItemBuilder>()
```

Comment:
We are supposed to use mock objects for anything that touches the real world (Database/Network/etc). If something is purely logical (like `FacilityListItemBuilder` whose responsibilty is to map items of one type to another) we should use real objects as much as possible.

Otherwise, it is possible that we change the implementation in production in a way that breaks the behaviour, but since we are using a mock object in the tests, they will continue to pass and we will never realise it.